packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,90 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.CheckboxField {
+    max-width: 100%;
+}
+
+.CheckboxField__label {
+    @include typography.generate(theme.$label-typography);
+
+    margin-bottom: theme.$spacing;
```

Comment:
This is not applied because `label` is in inline display mode.

Reply:
I would remove this space, as its there even without message.

Reply:
Fixed.